transpiler/languages/c_plus_plus.py

```diff
@@ -0,0 +1,42 @@
+code = {
+    # Output
+    'print': {
+        'entities': ['to_print'],
+        'code': 'cout << {to_print};'
```

Comment:
In C++, for `cout`, `cin`, `vector`, `string` etc. use `std::` prefix to specify the namespace (eg. `std::cout`). If we simply use `using namespace std;` at the beginning it may cause problems in the future ("namespace pollution").